app/services/analytics_events.rb

```diff
@@ -4116,6 +4116,20 @@ def piv_cac_update_name_submitted(
     )
   end
 
+  # Tracks when user visits piv cac recommended
+  def piv_cac_recommended_visited
+    track_event(:piv_cac_recommended_visited)
+  end
+
+  # @param [String] action what action user made
+  # Tracks when user submits an action on Piv Cac recommended page
+  def piv_cac_recommended(action: nil, **_exra)
+    track_event(
+      :piv_cac_recommended,
+      action: action,
```

Comment:
1. Typo on `extra`
2. Any reason we shouldn't log those extra? Related: #10262

```suggestion
  def piv_cac_recommended(action: nil, **extra)
    track_event(
      :piv_cac_recommended,
      action: action,
      **extra,
```